mpt-witness-generator/witness/gen_witness_from_infura_blockchain_test.go

```diff
@@ -769,6 +769,9 @@ func TestExtensionDeletedOneKeyByteSel1(t *testing.T) {
 }
 
 func TestExtensionOneKeyByteSel2(t *testing.T) {
+	// TODO re-enable this test
```

Comment:
Add issue?

Reply:
I tested it now and it works. I can't remember, when and why it was disabled, but probably because it's much slower than the other tests. So you can reenable it here or I will do it in the next PR.